src/unit/format.js

```diff
@@ -0,0 +1,49 @@
+define([
+	"../common/format-message"
+], function( formatMessage ) {
+
+/**
+ * format( value, unit, pluralGenerator )
+ *
+ * @value [Number]
+ *
+ * @unitProperies [Object]: localized unit data from cldr.
+ *
+ * @pluralGenerator [Object]: A pluralGenerator from Globalize.pluralGenerator.
+ *
+ * TODO pass along numberFormatter
```

Comment:
I don't think it's too much trouble to pass a custom number formatter, I'll take a look at that today.